base/linalg/dense.jl

```diff
@@ -5,6 +5,7 @@
 ## BLAS cutoff threshold constants
 
 const SCAL_CUTOFF = 2048
+const TRIA_CUTOFF = 2048
```

Comment:
Would be good to have in base a function to get the size of the cache... In the meantime those arbitrary numbers may be fine.  
Also, the cutoff should depend on the `sizeof` of the (elements of the) matrice, no? So this could be better to divide a threshold by this `sizeof`.